### DISCUSSION TITLE
Windows 11 support?

### DISCUSSION THREAD

**Discussion by hoelderlin** [t1m0thyj/WinDynamicDesktop#350] [Q&A] [answered]
Hi, anyone has tried WinDynamicDesktop in Windows 11?

**Comment by emberaholdon** [comment, marked as answer]
Hi,

I upgraded my Win 10 Pro on my work laptop which is a DELL Latitude 5401. So far no problems with WDD. I'm using the Mojave theme with hidden tray icon.

**Reply by hoelderlin** [reply]
So, you just upgraded and WDD kept working like in Win 10? Good news!

**Reply by emberaholdon** [reply]
Pretty much. I use it together with [Armin Osaj's Auto Dark Mode](https://github.com/Armin2208/Windows-Auto-Night-Mode/releases) and they're both working fine. I wish Microsoft has just built-in the automatic theme and wallpaper switching like in macOS. Fingers crossed for RTM. :)

**Reply by hoelderlin** [reply]
No, I prefer that developers like Tim get some profit from this ;) 
Look in my case, WDD switches automatically to dark mode. Your doesn't?

**Reply by emberaholdon** [reply]
I didn't know WDD has theme switching capabilities. To be clear, I'm talking about Windows dark and light theme which can be set in Settings - Personalization - Colors.

**Reply by hoelderlin** [reply]
Yes, WDD has theme switching capabilities! You have to install two scripts: ChangeWindowsAppTheme.ps and ChangeWindowsSystemTheme.ps and enable PowerScripts under More Options. Ah, this only works in the github version; the version from the MS Store does not work.

See it here: https://github.com/t1m0thyj/WDD-scripts/tree/master/stable#readme

**Reply by wkdk** [reply]
> Yes, WDD has theme switching capabilities! You have to install two scripts: ChangeWindowsAppTheme.ps and ChangeWindowsSystemTheme.ps and enable PowerScripts under More Options. Ah, this only works in the github version; the version from the MS Store does not work.
> 
> See it here: https://github.com/t1m0thyj/WDD-scripts/tree/master/stable#readme

Actually and surprisingly, **all scripts work in Microsoft Store version also.**

**Comment by Cmdr-Keene** [comment]
I'm using Windows 11 and can confirm that WDD switches themes properly using the powershell script feature.

I realized that you _could_ roll your own WDD with powershell scripts and scheduled tasks in Windows, but c'mon, WDD is excellent software and great, I am evangelizing the heck out of it now.  I've created a few themes and submitted them to the community site now as well.  I'm surprised Windows doesn't include this in their OS, but it's awesome that some devs have picked in to fill the gap.

**Comment by wkdk** [comment]
I have clean installed Windows 11 and Microsoft Store version of WDD works perfectly, also all scripts work too (even if in Store version).

**Comment by TechStudent10** [comment]
So far, WDD is working in Windows 11 for me. I am using Windows 11 Version 21H2 (OS Build 22000.318).
